loader/readsource.go

```diff
@@ -44,5 +46,12 @@ func ReadSource(src, pwd string, filesystems map[string]afero.Fs, stdin io.Reade
 	if err != nil {
 		return nil, err
 	}
-	return Load(filesystems, srcURL, src)
+	result, err := Load(filesystems, srcURL, src)
+	var noSchemeError noSchemeRemoteModuleResolutionError
+	if errors.As(err, &noSchemeError) {
+		// TODO maybe try to wrap the original error here as well, without butchering the message
+		return nil, fmt.Errorf(nothingWorkedLoadedMsg, noSchemeError.moduleSpecifier, noSchemeError.err)
```

Comment:
I'm using the latest `golangci-lint` version locally, and this line fails with `goerr113  err113: do not define dynamic errors, use wrapped static errors instead`, which is from a new linter in `v1.26.0`, and we should probably follow it. :)

I think it might be just a matter of using `%w` in the format string, and then we could also remove the `Unwrap()` method here.

Reply:
As far as I understood it - I will need to define a new error type as I also have other parameters ... I also really wanted this to be a quick fix. 
And this whole code probably needs additional error types, but again ... I wanted this to be 5 minutes fix so we stop getting all the module resolution problems. 

This *particular* really long error message is only if you managed to not write the correct filename/url you want to execute, at which point, again, because of us doing schemeless remote module resolution we can have different errors, for things totally unrelated to what you meant to do. 

If we didn't I could just write "file something.js wasn't found, if you are using docker ..." or "https://something.com returner error: ..." and be done with it...